go/arrow/csv/reader_test.go

```diff
@@ -61,6 +62,11 @@ func Example() {
 		n++
 	}
 
+	err := r.Err()
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
I'd say add a comment here to be more informative and explicit about the error handling

Reply:
I have added a comment describing why we are checking the error.